src/zenml/zen_stores/migrations/versions/d02b3d3464cf_add_orchestrator_run_id_column.py

```diff
@@ -0,0 +1,40 @@
+"""Add orchestrator_run_id column [d02b3d3464cf].
+
+Revision ID: d02b3d3464cf
+Revises: ccd68b7825ae
+Create Date: 2022-10-26 16:50:44.965578
+
+"""
+import sqlalchemy as sa
+import sqlmodel
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "d02b3d3464cf"
+down_revision = "ccd68b7825ae"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    """Upgrade database schema and/or data, creating a new revision."""
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table("pipelinerunschema", schema=None) as batch_op:
+        batch_op.add_column(
+            sa.Column(
+                "orchestrator_run_id",
+                sqlmodel.sql.sqltypes.AutoString(),
+                nullable=False,
```

Comment:
This breaks existing DBs if nullable is False. I would propose to simply allow it to be None for backwards compatibility. If you really want to enforce that it is not None for all entries, you need to a migration in three steps: 1) add column with nullable True 2) run SQL command to define values for all null columns 3) update column to set nullable False

Reply:
See how we added status here for example: https://github.com/zenml-io/zenml/pull/977/files#diff-a276a0accba80e17b01aa2f049ee0affeed819b35743017dbfbc23fd931df277

Reply:
Actually this is outdated already, I've changed it to be `nullable=True`. Do I still need a three step approach in this case, or can I leave the migration unchanged otherwise?

Reply:
In that case you can leave it unchanged 👍